packages/fxa-settings/src/components/UnitRowRecoveryKey/index.test.tsx

```diff
@@ -3,9 +3,27 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import React from 'react';
-import { screen } from '@testing-library/react';
+import { screen, fireEvent, act } from '@testing-library/react';
 import UnitRowRecoveryKey from '.';
-import { renderWithRouter, MockedCache } from '../../models/_mocks';
+import {
+  renderWithRouter,
+  MockedCache,
+  MOCK_ACCOUNT,
+} from '../../models/_mocks';
+import { GET_ACCOUNT } from 'fxa-settings/src/models';
+
+const mockAccountQuery = (modifications: { [key: string]: any } = {}) => {
```

Comment:
This might be worth moving into `_mocks` since we do this frequently. It would also be nice if `any` could somehow reflect the account interface 🤔

Reply:
> It would also be nice if any could somehow reflect the account interface

I tried something like `{ [key: keyof Account]: Account[typeof key] }` but I got "An index signature parameter type cannot be a union type. Consider using a mapped object type instead." and my TypeScript bottoms out around there.